src/libstore/path.cc

```diff
@@ -63,7 +63,17 @@ StorePath StorePath::random(std::string_view name)
 
 StorePath StoreDirConfig::parseStorePath(std::string_view path) const
 {
-    auto p = canonPath(std::string(path));
+    // On Windows, `/nix/store` is not a canonical path. More broadly it
+    // is unclear whether this function should be using the native
+    // notion of a canonical path at all. It makes to talk about remote
+    // stores whose store dir is the "other" kind of path.
```

Comment:
The last sentence of this comment doesn't make sense to me.

Reply:
Reworded